en_US/durability/management.md

```diff
@@ -1,91 +1,79 @@
-| `durable_storage.<DS>.layout`             | This sub-tree contains the parameters that control how EMQX lays out the data on disk. See description below.      |
+#### Local Write Buffer Configuration
 
-### Local write buffer configuration
+To maximize the throughput, EMQX writes MQTT messages from the clients to the durable storage in batches. Batching is configured using the following parameters under `durable_storage.<DS>.layout` configuration sub-tree:
 
-In order to maximize the throughput, EMQX writes MQTT messages from the clients to the durable storage in batches.
-Batching is configured using the following parameters under `durable_storage.<DS>.layout` configuration sub-tree:
+| Parameter        | Description                                                  |
+| ---------------- | ------------------------------------------------------------ |
+| `max_items`      | The buffer is flushed when its size reaches this value.      |
+| `flush_interval` | The buffer is also flushed at this interval, provided it contains at least one message. |
 
-| Parameter        | Description                                                                                            |
-|------------------|--------------------------------------------------------------------------------------------------------|
-| `max_items`      | The buffer is flushed when its size reaches this value                                                 |
-| `flush_interval` | Additionally, the buffer is flushed every flush interval, as long as it contains at least one message. |
+#### Storage Layout Configuration
 
-### Storage layout configuration
+Storage layout determines how EMQX organizes data on disk. Setting `durable_storage.<DS>.layout.type` parameter can change the layout used by the new [generations](./durability_introduction.html#generation). This change does not affect existing generations. The configuration of each layout type varies and is contained under the `durable_storage.<DS>.layout` sub-tree. Currently, the `wildcard_optimized` layout type is available.
 
-Storage layout determines how EMQX organizes data on disk.
-Layout used by the new [generations](./durability_introduction.html#generation) can be changed by setting `durable_storage.<DS>.layout.type` parameter.
-It does not affect the generations that has been already created.
-Currently, the following layouts are available:
+##### Configuration of `wildcard_optimized` Layout Type
 
-- `wildcard_optimized`
+The `wildcard_optimized` layout is aimed to optimize wildcard subscriptions across a vast array of topics. It achieves this by autonomously accumulating knowledge about topic structures over time. Leveraging a lightweight machine learning algorithm, it predicts the wildcard topic filters that clients are likely to subscribe to. Subsequently, it organizes these topics into a unified stream, facilitating efficient consumption in a single batch.
```

Comment:
> facilitating efficient consumption in a single batch.

"facilitating efficient consumption in a single batch" -> "allowing efficient consumption in a single sweep". The original phrasing was imprecise too, because size of the batch is controlled by the client.

Reply:
Thank you for the correction! I have updated it.